spidy/tests/test_try.py

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env python
+
+import unittest
+import rostest
+import rospy
+from std_msgs.msg import Float64
+
+
+
+bat=0
+class MyTestCase(unittest.TestCase):
+
+    def callback(self, data):
+        bat=data.data
```

Comment:
write good unit and integration tests. Now it is nothing.

Reply:
changed both testing files to more reasonable testing technique